docs/workflows/scripting.rst

```diff
@@ -5,6 +5,8 @@ Each workflow renders bash scripts that allow the developers to ensure the conti
 the instructions. These scripts may also be helpful to users as a basis for their own scripts. All
 of the scripts can be found at https://github.com/pulp/pulp_rpm/tree/master/docs/_scripts/
 
+In some script you could find few conditional, there are only for clean run of correctness check.
```

Comment:
What does this mean?

Reply:
To be aware why there is some conditionals (if/else) in the scripts.
E.g. https://github.com/pulp/pulp_rpm/pull/1661/files#diff-15a453c39058db52065840dc81482df5R4

Reply:
I'll try to explain better: as these scripts appear in docs and reuse parts of code as repository creation and with this PR we want to ensure when we run both docs_check_[sync|upload].sh it won't fail (e.g. by same repository name) I wanted to warn users that in the doc scripts are some if/else blocks which is not important for the workflow but useful for script check itself.

Reply:
These scripts might be helpful to users as a basis for their own scripts. In some scripts you will find a few conditional blocks. These are only used for a clean run of the correctness check. All
of the scripts can be found at https://github.com/pulp/pulp_rpm/tree/master/docs/_scripts/